types/n3/index.d.ts

```diff
@@ -130,6 +130,24 @@ export interface BlankTriple<Q extends RDF.BaseQuad = RDF.Quad> {
     object: Q['object'];
 }
 
+export interface Token {
+    type: string;
+    value?: string;
+    line: number;
+}
```

Comment:
```suggestion
    prefix?: string;
}
```

Tokens can also have a prefix field: https://github.com/rdfjs/N3.js/blob/89f0daf748bf2d6ca194b37e58733798346a88e0/src/N3Lexer.js#L348